dolweb/downloads/templates/downloads-index.html

```diff
@@ -87,6 +87,15 @@
 </table>
 </div>
 
+<div id="Other-Linux">
+    <h1>{% trans "Older Ubuntu (< 15.04) and other Linux distros" %}</h1>
+    <p>{% blocktrans %}Ubuntu 14.10 and older users can install a PPA 
+    for development and stable versions of Dolphin here: <a href=https://wiki.dolphin-emu.org/index.php?title=Installing_Dolphin#Ubuntu>Installing Dolphin</a>
+    {% endblocktrans %}</p>
+    <p>{% blocktrans %}Other Linux distro users can look here to compile Dolphin: <a href=https://wiki.dolphin-emu.org/index.php?title=Building_Dolphin_on_Linux>Building Dolphin on Linux</a>
```

Comment:
I'm not a native english speaker, but "Other Linux distro users" sounds weird to me. First of all, s/distro/distributions/, no need to abbreviate in this context. Second of all, I would turn it around like "Users of other Linux distributions".

But again, ignore if I'm wrong here.